libs/clients/blood/src/lib/dtos/bloodTypes.dto.ts

```
@@ -0,0 +1,31 @@
+}
+
+export const mapBloodTypeDto = (data: PatientBlood): BloodTypeDto | null => {
+  if (!data.nationalId || !data.type) {
+    return null
+  }
+  return {
+    nationalId: data.nationalId,
+    type: data.type,
+    registered: data.registerd ? data.registerd.toISOString() : undefined,
+    bloodInfo:
+      data.bloodInfo && data.bloodInfo.descripton
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)